### PR TITLE
smaller compposer

### PR DESCRIPTION
**Pull request by cyberhck**
smaller composer uses alpine's php7

### CODE REVIEW

**Comment by cyberhck** [comment]
@sergeyklay any review on this one?

File: composer/7/Dockerfile [hunk 1]
```diff
@@ -5,19 +5,13 @@
 #
 
 # Pull base image
-FROM phalconphp/php:7
+FROM phalconphp/php:alpine-php7-min
 
 MAINTAINER Serghei Iakovlev <serghei@phalconphp.com>
+ENV COMPOSER_ALLOW_SUPERUSER 1
+ENV COMPOSER_HOME /composer
```
Comment (sergeyklay) [inline composer/7/Dockerfile]: could you please use single docker layer?
```
ENV COMPOSER_ALLOW_SUPERUSER=1 \
    COMPOSER_HOME=/composer
```

File: composer/7/Dockerfile [hunk 2]
```diff
@@ -5,19 +5,13 @@
...
 
-# Set environment variables
-ENV COMPOSER_HOME=/root/composer \
-    PATH=$COMPOSER_HOME/vendor/bin:$PATH \
```
Comment (sergeyklay) [inline composer/7/Dockerfile]: add `PATH=$COMPOSER_HOME/vendor/bin:$PATH` back please

File: composer/7/Dockerfile [hunk 3]
```diff
@@ -30,5 +24,4 @@ VOLUME ["/app"]
 WORKDIR /app
 
 # Set up the command arguments
-CMD ["-"]
-ENTRYPOINT ["composer", "--ansi"]
```
Comment (sergeyklay) [inline composer/7/Dockerfile]: could you explain this a bit more?

File: composer/7/Dockerfile (hunk 1 above)
Comment (cyberhck) [inline composer/7/Dockerfile]: will do this tonight :)

File: composer/7/Dockerfile (hunk 2 above)
Comment (cyberhck) [inline composer/7/Dockerfile]: do we really need that? I don't think we do, composer's executable is already available, and composer's global installations aren't meant to be executed inside a docker container. According to official composer, we should run like this:
```
docker run --rm --interactive --tty \
    --volume $PWD:/app \
    composer install
```

File: composer/7/Dockerfile (hunk 3 above)
Comment (cyberhck) [inline composer/7/Dockerfile]: `CMD ["composer"]` is all we need, we don't need anything else.

File: composer/7/Dockerfile (hunk 1 above)
Comment (cyberhck) [inline composer/7/Dockerfile]: done :)

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**